### ISSUE TITLE
Crashes in node

### ISSUE THREAD

**Issue by rutchkiwi** [derhuerst/pev2-cli#3]
Hi! I'm on a mac with ventura 13.4.1. When trying to run any command, I get a cryptic error from deep inside node:

```
❯ pev2 -o ~/dev/0slask/packing-perf.sql /tmp/pev_query_plan.json
node[31083]: ../src/node_url.cc:270:static void node::url::BindingData::Update(const FunctionCallbackInfo<v8::Value> &): Assertion `out' failed.
 1: 0x10318a60f node::Abort() [/usr/local/Cellar/node/20.4.0/bin/node]
 2: 0x10318a5fa node::Abort() [/usr/local/Cellar/node/20.4.0/bin/node]
 3: 0x103231d3f node::url::BindingData::Update(v8::FunctionCallbackInfo<v8::Value> const&) [/usr/local/Cellar/node/20.4.0/bin/node]
 4: 0x10334d982 v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [/usr/local/Cellar/node/20.4.0/bin/node]
 5: 0x10334d513 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, unsigned long*, int) [/usr/local/Cellar/node/20.4.0/bin/node]
 6: 0x10334d23b v8::internal::Builtin_HandleApiCall(int, unsigned long*, v8::internal::Isolate*) [/usr/local/Cellar/node/20.4.0/bin/node]
 7: 0x102fb59f6 Builtins_CEntry_Return1_ArgvOnStack_BuiltinExit [/usr/local/Cellar/node/20.4.0/bin/node]
 8: 0x102f274dc Builtins_InterpreterEntryTrampoline [/usr/local/Cellar/node/20.4.0/bin/node]
zsh: abort      pev2 -o ~/dev/0slask/packing-perf.sql /tmp/pev_query_plan.json
```

I really have no idea where to start fixing this, but I'd appreciate any help!

**Comment by derhuerst** [comment]
Please provide the version of `pev2-cli` you're using, as well as the Node.js version.

**Comment by rutchkiwi** [comment]
Of course:
pev2-cli v1.20.0
v20.4.0

**Comment by derhuerst** [comment]
This looks like it happens due to https://github.com/nodejs/node/issues/49650.

**Comment by derhuerst** [comment]
Published a workaround as `pev2-cli@1.20.1`. Can you try it?

**Comment by rutchkiwi** [comment]
It works perfectly. Thank you so much! :) This tool really speeds up my SQL optimisation workflow.
